modal.js

```diff
@@ -0,0 +1,122 @@
+const gridContainer = document.querySelector('.grid-container');
+
+const projects = [
+  {
+    title: 'Tonic',
+    description: 'A daily selection of privately personalized reads no accounts or sign-ups required.',
+    imageLinks: './img/snapshoot-portfolio.png',
+    technologies: ['html', 'css', 'javascript'],
+    role: 'Back End Dev',
+    year: '2022',
+    company: 'Canopy',
+    live: 'https://dinma-faith.github.io/Dinma-Portfolio.github.io/',
+    source: 'https://github.com/Dinma-Faith/Dinma-Portfolio',
+  },
```

Comment:
- [ ] Currently, your design is not matched with the Figma design (Desktop version). See the below screenshot about the space on the left, right, and bottom also. Please fix that.

- [ ] When it popup, the background should be blurred. Please work on that for both mobile and desktop versions. 

Your design             |  Figma Design
:-------------------------:|:-------------------------:
![d2](https://user-images.githubusercontent.com/70483835/162511616-7b877d5d-205b-45f8-8151-88777645e690.png) | ![d1](https://user-images.githubusercontent.com/70483835/162511353-c77ae024-dfaf-45ef-aba2-7b44fb20812a.png)

- [ ] When clicking on the see project button,  the navbar is still there. Check the image below and fix it.

Your design             |  Figma Design
:-------------------------:|:-------------------------:
![m1](https://user-images.githubusercontent.com/70483835/162515982-282dfe24-be0e-4e66-b29d-1a6e1d00e5c3.png) | ![m2](https://user-images.githubusercontent.com/70483835/162516009-e8ef951a-4208-4a6d-96e5-632631261bc2.png)